docs/rest_api.md

```diff
@@ -135,27 +135,23 @@ The result must contain the following fields:
 * "channels", description of legal channel identifiers for this network.
   This is a list of channel identifiers, with possible wildcards. The
   format of these is driver specific, see below.
-* "creator", the name of the project which created the network, or
+* "owner", the name of the project which created the network, or
   "admin", if it was created by an administrator.
-
-The result may also contain the following fields:
-
-* "access" -- if this is present, it is the name of the project which
-  has access to the network. Otherwise, the network is public.
+* "access", a list of projects that have access to the network or 'None' if the network is public
```

Comment:
Right, but this is the API docs, so we should be talking in terms of what ends up on the wire. Python `None` gets mapped to JSON `null`.